src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -148,10 +151,70 @@ def init_db():
     """
     # SyncD database connector. THIS MUST BE INITIALIZED ON A PER-THREAD BASIS.
     # Redis PubSub objects (such as those within swsssdk) are NOT thread-safe.
-    db_conn = SonicV2Connector(**redis_kwargs)
+    db_conn = SonicV2Connector(**redis_kwargs) 
 
     return db_conn
 
+def init_namespace_dbs():
+    db_conn= []
+    # Global db connector
+    SonicDBConfig.load_sonic_global_db_config()
+    for namespace in SonicDBConfig.get_ns_list():
+        db = SonicV2Connector(use_unix_socket_path=True, namespace=namespace)
+        db_conn.append(db)
+
+    return db_conn
+
+def connect_all_namespace_dbs(all_ns_db, db_id):
+    for db_conn in all_ns_db:
+        db_conn.connect(db_id)
+
+def get_keys_from_all_namespace_dbs(all_ns_db, db_id, pattern='*'):
```

Comment:
>db_id [](start = 47, length = 5)

always use db_name?
I am worried you are not using latest code in sonic-py-swsssdk repo.

Reply:
Fixed